src/test/scala/org/scalacheck/PropSpecification.scala

```diff
@@ -24,7 +24,7 @@ object PropSpecification extends Properties("Prop") {
 
   property("Prop.==> undecided") = forAll { p1: Prop =>
     val g = oneOf(falsified,undecided)
-    forAll(g) { p2 =>
+    forAllNoShrink(g) { p2 =>
```

Comment:
Now that you've defined `forAllShrink(Gen)`, you could use it in the test suite.  But it would be better to just minimize the changes in the PR, by keeping `forAll(Gen)` in the tests.  The deprecation warnings for `forAll(Gen) are non-fatal in the test suite, anyway.